development/request_guidelines.md

```diff
@@ -1,3 +1,21 @@
+# Pull Request Expectations
+
+We have a set of norms around pull requests and merging of code. Most of these have evolved in an informal way among development teams or during our retrospectives while discussing things we could improve. There are exceptions to many rules, but this document attempts to identify our baseline expectations.
+
+## For Submitters
+
+- When containers are modified as a part of a pull request manifests should be updated to use those containers. Alternatively, instructions may be provided to inform reviewers how to deploy the change.
+- All non-trivial pull requests are expected to have a set of testing manual instructions with them. The more information you can provide reviewers about how to verify the change the more likely you are to get good reviews.
+- Pull requests should be merged by the person who submitted them. We leave them open for the submitter so they may do minor cleanup before merging and control the order of merges if they have multiple PRs open.
+- Pull requests should never be merged without a review. We require 1 review for smaller pull requests and at least 2 for larger pull requests. There's no defined line between "small" and "large" pull requests, and that is left to the submitter's discretion. The more reviews you get the less likely you are to be introducing bugs or nonworking code. As a general rule, we value stability over speed when it comes to introducing features.
+- All repositories are expected to use our [build tools](https://github.com/drud/build-tools).
+
+## For Reviewers
```

Comment:
I see nothing here regarding the expectation of tests and documentation. I think that should be mandatory that documentation changes are included alongside and it requires an approved exception to merge without these in place.

Reply:
I disagree on this one. The goal of this document is to accurately capture what our current development practices are.

I think always pushing tests and documentation alongside a pull request is a fantastic goal, but I don't agree that we always do so now. It's pretty common for us to do follow-ups on those items I think.

Reply:
I added, "It is strongly suggested that automated tests and documentation updates are included in pull requests." to reflect these aspirations :)